bittensor/extrinsics/staking.py

```diff
@@ -649,4 +650,20 @@ def normalize_children_and_proportions(
     """
     total = sum(prop for prop, _ in children)
     u64_max = 2**64 - 1
-    return [(int(prop * u64_max / total), child) for prop, child in children]
+    normalized_children = [
+        (int(floor(prop * (u64_max - 1) / total)), child) for prop, child in children
+    ]
+    sum_norm = sum(prop for prop, _ in normalized_children)
+
+    # if the sum is more, subtract the excess from the first child
+    if sum_norm > u64_max:
+        if abs(sum_norm - u64_max) > 10:
+            raise ValueError(
+                "The sum of normalized proportions is out of the acceptable range."
+            )
+        normalized_children[0] = (
+            normalized_children[0][0] - (sum_norm - (u64_max - 1)),
+            normalized_children[0][1],
```

Comment:
It would be useful to understand what such indexes mean. Pls add a comment.

Reply:
There is a comment on line 658. 
It adjusts the first child +- 10 to make sure we dont go over u64 max.